packages/Tokens/src/tokens.yaml

```diff
@@ -50,14 +50,14 @@ color:
   orange-disabled: $color--orange-lighten-20
   orange-highlight: '#e1624b'
   #
-  purple: '#53529f'
-  purple-darken-10: darken($color--purple, 10)
-  purple-darken-20: darken($color--purple, 20)
-  purple-lighten-10: desaturate(lighten($color--purple, 10), 5)
-  purple-lighten-20: desaturate(lighten($color--purple, 20), 5)
-  purple-lighten-30: desaturate(lighten($color--purple, 30), 5)
-  purple-lighten-40: desaturate(lighten($color--purple, 40), 5)
-  purple-lighten-50: desaturate(lighten($color--purple, 50), 5)
+  galvanize: '#785cba'
+  galvanize-darken-10: darken($color--galvanize, 10)
+  galvanize-darken-20: darken($color--galvanize, 20)
+  galvanize-lighten-10: desaturate(lighten($color--galvanize, 10), 5)
+  galvanize-lighten-20: desaturate(lighten($color--galvanize, 20), 5)
+  galvanize-lighten-30: desaturate(lighten($color--galvanize, 30), 5)
+  galvanize-lighten-40: desaturate(lighten($color--galvanize, 40), 5)
+  galvanize-lighten-44: desaturate(lighten($color--galvanize, 44), 5)
```

Comment:
Also, 44 is a very arbitrary value. We need to treat this as a design system conversation and adjust the colors if absolutely necessary.

Reply:
@johnpeele wouldn't be more efficient if you provided us with the exact number of colours that we need to support? to me seems too many colours, I'm not entirely sure the need for supporting all these colours, but that's just my opinion.

Reply:
@nahumzs I agree 100%, but unfortunately we are not ready to redo our color palettes.